src/style.css

```diff
@@ -1,3 +1,74 @@
 body {
   margin: 0;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+
+  --link-hover-color: #00acdc;
+  --link-active-color: var(--link-hover-color);
+  --header-max-width: 1200px;
+  --header-min-width: 1024px;
+  --header-bckgrd-color: #fff;
+  --header-line-height: 60px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  max-width: var(--header-max-width);
+  min-width: var(--header-min-width);
+  margin: 0 auto;
+  background-color: var(--header-bckgrd-color);
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  margin: auto 0 auto 50px;
+}
+
+.logo__img {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  margin: 0 50px 0 0;
```

Comment:
For the future, you can`t determinate external position of BEM block, but you can do it inside BEM element of parent BEM block